interface/batchcom/batchcom.php

```diff
@@ -24,280 +29,217 @@
 include_once("../../library/acl.inc");
 include_once("batchcom.inc.php");
 
-// gacl control
-$thisauth = acl_check('admin', 'batchcom');
+use OpenEMR\Core\Header;
 
-if (!$thisauth) {
-  echo "<html>\n<body>\n";
-  echo "<p>".xl('You are not authorized for this.','','','</p>')."\n";
-  echo "</body>\n</html>\n";
-  exit();
- }
+if (!acl_check('admin', 'batchcom')) {
+    echo "<html>\n<body>\n";
+    echo "<h1>".xl('You are not authorized for this.','','','</h1>')."\n";
+    echo "</body>\n</html>";
+    exit();
+}
 
 // menu arrays (done this way so it's easier to validate input on validate selections)
-$choices=Array (xl('CSV File'),xl('Email'),xl('Phone call list'));
-$gender=Array (xl('Any'),xl('Male'),xl('Female'));
-$hipaa=Array (xl('NO'),xl('YES'));
-$sort_by=Array (xl('Zip Code')=>'patient_data.postal_code',xl('Last Name')=>'patient_data.lname',xl('Appointment Date')=>'last_ap' );
+$process_choices = array (xl('Download CSV File'),xl('Send Emails'),xl('Phone call list'));
+$gender_choices = array (xl('Any'),xl('Male'),xl('Female'));
+$hipaa_choices = array (xl('No'),xl('Yes'));
+$sort_by_choices = array (xl('Zip Code')=>'patient_data.postal_code',xl('Last Name')=>'patient_data.lname',xl('Appointment Date')=>'last_ap' );
```

Comment:
I am willing to bet last_ap above should be last_appt since last_ap is not anywhere in the query whereas last_appt is